sql/src/main/java/io/crate/execution/engine/collect/stats/JobsLogs.java

```diff
@@ -119,20 +123,31 @@ public void logExecutionEnd(UUID jobId, @Nullable String errorMessage) {
         }
         LogSink<JobContextLog> jobContextLogs = jobsLog.get();
         JobContextLog contextLog = new JobContextLog(jobContext, errorMessage);
-        histogram.recordValue(contextLog.ended() - contextLog.started());
+        addToHistogram(contextLog);
         jobContextLogs.add(contextLog);
     }
 
+    private void addToHistogram(JobContextLog log) {
+        StatementClassifier.Classification classification = log.classification();
```

Comment:
would it make sense to extract the logic here, the maps and constants into a `ClassifiedHistograms` class? For separation of concerns and making it easier to test things in isolation/smaller units